src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -97,42 +97,42 @@ public void parse_optionalFieldsMissing_success() {
         Person expectedPersonWithoutPhone = new PersonBuilder(AMY).withoutPhone().build();
         assertParseSuccess(parser, NAME_DESC_AMY + EMAIL_DESC_AMY + ADDRESS_DESC_AMY
                         + CLIENTSOURCE_DESC_FRIEND + NOTE_DESC_CAT + PRIORITY_DESC_AMY,
-                new AddCommand(expectedPersonWithoutPhone));
+                new AddCommand(expectedPersonWithoutPhone, null));
```

Comment:
Perhaps we should have Policy here, as we are testing one field at a time. Same for rest of optional fields missing